modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -104,13 +145,15 @@ public function add_iframed_body_class( $classes ) {
 	/**
 	 * Checks whether this is a whitelisted iframe request.
 	 *
+	 * @param string $nonce Optional. Nonce to verify. Default $_GET['frame-nonce'].
 	 * @return bool
 	 */
-	public function framing_allowed() {
-		$verified = $this->verify_frame_nonce(
-			$_GET['frame-nonce'], // phpcs:ignore WordPress.Security.NonceVerification
-			'frame-' . Jetpack_Options::get_option( 'id' )
-		);
+	public function framing_allowed( $nonce = '' ) {
```

Comment:
Since we're the only consumers of this method, can we just make it require a nonce, rather than the backup value dance below (that may or may not exist)? That way we can have `framing_allowed` always called with a param, and just skip the empty check.

Reply:
I'm not sure how I feel about f196f4b, but if it gets this approved faster…